src/test/java/api/support/fakes/FakeOkapi.java

```diff
@@ -199,6 +199,8 @@ public void start(Future<Void> startFuture) throws IOException {
       .withDisallowedProperties("pickupServicePoint", "loan", "deliveryAddress")
       .withRecordConstraint(this::requestHasSamePosition)
       .withChangeMetadata()
+      .withBatchUpdate("/request-storage-batch/requests")
+      .withBatchUpdatePreProcessor(this::resetPositionsBeforeBatchUpdate)
```

Comment:
We can, but I think it will be more challenging than the pre-processor, because we will have to consider a rollback mechanism, because data will be inconsistent.

Reply:
Ok, I'll leave that to your judgement

Reply:
Ok, I'll leave that to your judgement